tests/work_queue.rs

```diff
@@ -0,0 +1,67 @@
+extern crate beanstalkd;
```

Comment:
Let's rename the this file. Looks like you copied `work_queue.rs` over and didn't rename it ☺️

Reply:
Yes, good catch.